subsys/net/openthread/Kconfig.defconfig

```diff
@@ -242,4 +242,10 @@ config OPENTHREAD_BLE_TCAT_THREAD_STACK_SIZE
 
 endif # OPENTHREAD_BLE_TCAT
 
+config OPENTHREAD_NUM_MESSAGE_BUFFERS
```

Comment:
Just a question.. Would it make sense to add: `depends (OPENTHREAD_SOURCES || (OPENTHREAD_LIBRARY && DYNAMIC_PLATFORM_MEM_MGMT..)`

Reply:
We want to support `DYNAMIC_PLATFORM_MEM_MGMT` by default in library as well soon, and precompiled values of these configs are copied on lib creation in `openthread_lib_configuration.txt`. as far as I remember there will be no warnings on values different than in lib config, so this change won't do anything for libs for now. dependency can be added but we need to make sure that it will be removed before next lib deployment

Reply:
We're overriding default value of this option defined in [zephyr](https://github.com/adigie/sdk-zephyr/blob/4f3ef0aa00b54dc19da82629c3402b2af149dfb5/subsys/net/l2/openthread/Kconfig#L264). If the condition is not met, we would get default from there. Adding `depends` here won't work.